stacks/api/function/packages/database/test/database.spec.ts

```diff
@@ -63,7 +74,7 @@ describe("database", () => {
     setEnvironment();
     const db = await database();
     expect(connectSpy).toHaveBeenCalledTimes(1);
-    expect(isConnectedSpy).toHaveBeenCalledTimes(1);
+    expect(isConnectedSpy).toHaveBeenCalled();
```

Comment:
Why did we need to make this change?

Reply:
isConnectedSpy could be called N times depending on the jasmine's seed number. So we don't care how many times it has been called. We just care if it was called.